forem/forem
# Z-index fix

## What type of PR is this? (check all applicable)


## Description

This PR fixes issues caused by recent update to our z-indexes in Crayons. The issue was sticky header being below sidebar contents on profile view and tag view.

## Related Tickets & Documents

- Closes #11202
- Related #11204 
- Related #11223 
- Related #11228 

## QA Instructions, Screenshots, Recordings

Check if sticky header is being overlapped by sidebars on profile view or a tag view.

## Added tests?

- [x] No, not needed

## Added to documentation?
